lib/msal-browser/src/network/XhrClient.ts

```diff
@@ -3,28 +3,30 @@
  * Licensed under the MIT License.
  */
```

Comment:
Why use xhr versus fetch?

Reply:
This is only used as a backup if the fetch client is not available in the window.

Reply:
cool!

Reply:
Can we polyfill fetch so we don't have to write this? 
cc @jasonnutter  if you have any experience with this

Reply:
@pkanher617 and I decided against that, so that we don't have to ship a polyfill, which comes with overhead that will only be needed on a minority of browsers. Applications can provide their own `fetch` polyfill or network client implementation if they would prefer not to use built-in XHR for IE.